File: 1.Check_Number_Positive_Or_Negative.java
// Importing the required package
import java.util.*;


// Defining a public class Main
public class Main{
  
// Main method of the program
public static void main(String[] args){
  
  
// Creating an object of Scanner class to read input from user
Scanner sc = new Scanner(System.in);
  
    // Reading an integer input from user and storing it in a variable called number
    int number = sc.nextInt();
    
    // Checking if the number is greater than zero
    if(number > 0){
      
        // Printing a message indicating that the number is positive
        System.out.println("Given Number Is Positive");
    }
  
    // Checking if the number is less than zero
    else if(number < 0){
      
	    // Printing a message indicating that the number is negative
	    System.out.println("Given Number Is Negative");
	}
  
	// If the number is not greater than or less than zero, it must be zero
	else{
    
	    // Printing a message indicating that the number is zero
	    System.out.println("The Number is Zero");
    }
  
  }
}

File: 1.Check_Number_Positive_Or_Negative.cpp
//This is used to include input/output header files.
#include <iostream>

//This is used to tell compiler that std:: namespace is used.
using namespace std;

//main function is the starting point of a C++ Program.
int main() {
  
  //int is a data-type which can hold integer data, n is the name of the variable used for storing data.
  
  //cin >> n is used to get user input from stdin tab.
  int n; cin >> n;
  
  //This is called nested-ternary operator which is used as replacement for if-else conditional statements.
  //Here (n > 0 ) is the first condition to be checked if we get true or 1 so after ? mark first statement which is "Positive" will execute
  //Otherwise second statement which is again a condition in this case (n < 0) will execute and again after ? mark first condition "Negative"
  //will print if it is true or 1 otherwise second statement "Neutral" will be printed.
  (n > 0 ) ? cout << "Positive" : (n < 0) ? cout << "Negative" : cout << "Neutral";
  
  //We can also use it like this for more readibility.
  //(n > 0 ) ? cout << "Positive" : (n < 0  ? cout << "Negative" : cout << "Neutral");
  
  //Used to tell that "No errors in the code".
  return 0;
  
}
